Logstash/logstash-input-azurewadtable/lib/logstash/inputs/azurewadtable.rb

```diff
@@ -52,14 +53,34 @@ def run(output_queue)
   def teardown
   end  
 
-  def process(output_queue)
-    @logger.debug(@last_timestamp)
+  def build_latent_query
```

Comment:
Will change `data_latency_seconds` to minutes.

Ideally only 1 minute at most would be a sufficient delay but I don't know if it is possible for VMs in a scale set to log data more than a minute late. This is why the delay is configurable to more than 1 minute. If this scenario is not possible, I'm happy to change the configuration to be a boolean.

For tables containing data for only a single VM, the delay should be unnecessary and users may like to have their data available ASAP, hence configurable to less than 1 minute (ie no delay).